docs/tutorial.md

```diff
@@ -737,3 +731,90 @@ Note that even though adding type annotations to .lua files makes it invalid
 Lua, you can still do so and load them from the Lua VM once the Teal package
 loader is installed by calling tl.loader().
 
+
+### Type definitions for third party libraries
+
+You can also create declaration files to annotate the types of third-party Lua
+modules, including C Lua modules. To do so, create a file with the .d.tl
+extension and require it as normal, i.e. `require("minetest")`.
+
+Types defined in this module will will be used as a source of type information 
+checking with `tl check`, even though the real Lua module will be loaded
+instead when requiring the module from Lua or `tl run`.
+
+If the third party library is "globally available", i.e. you do not explicitly
```

Comment:
```suggestion
If the third party library is "globally available" in your execution environment, i.e., if you do not explicitly
```

Reply:
I originally had this, not sure why I changed it, but if the preference is this way then 👍 from me.